### DISCUSSION TITLE
Welcome to auto-gpt-webui Discussions!

### DISCUSSION THREAD

**Discussion by choephix** [choephix/auto-gpt-webui#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0xVibez** [comment]
Getting this response when attempting to start AI on frontend
backend:start:     import requests
backend:start: ModuleNotFoundError: No module named 'requests'
backend:start:
